### DISCUSSION TITLE
iOS Terminal emulator that supports sc-im?

### DISCUSSION THREAD

**Discussion by ertect** [andmarti1424/sc-im#649] [Q&A] [answered]
I'd like to edit my daily habits spreadsheet while away from home, but unfortunately I have an iPhone, so I'm wondering if anyone has any ideas. Are terminal emulator apps that support sc-im? I can't find one because they usually say something like "impossible dependencies" when I try to install sc-im on those apps.

**Comment by 1dancook** [comment, marked as answer]
[iSH](https://github.com/ish-app/ish) can run it

**Reply by ertect** [reply]
Yep, just had to add these repositories:

http://dl-cdn.alpinelinux.org/alpine/v3.13/main
http://dl-cdn.alpinelinux.org/alpine/v3.13/community
http://dl-cdn.alpinelinux.org/alpine/edge/main
http://dl-cdn.alpinelinux.org/alpine/edge/community
http://dl-cdn.alpinelinux.org/alpine/edge/testing
http://dl-cdn.alpinelinux.org/alpine/latest-stable/main
http://dl-cdn.alpinelinux.org/alpine/latest-stable/community

[Source](https://github.com/ish-app/ish/issues/1217#issue-777669672)
